next.config.js

```diff
@@ -4,6 +4,10 @@ const { codecovWebpackPlugin } = require('@codecov/webpack-plugin');
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  serverRuntimeConfig: {
```

Comment:
The default size limit is already `1MB` and this is not the correct syntax to set it according to the docs and [this line](https://github.com/vercel/next.js/blob/canary/packages/next/src/server/config-shared.ts#L183)

The body size limit should live under `experimental.bodySizeLimit`